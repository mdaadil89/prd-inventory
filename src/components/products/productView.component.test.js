import React from 'react'
import {mount, shallow} from 'enzyme'
import ProductView from './productView.component'
import toJson from 'enzyme-to-json';


describe('AllProductsPage Snapshot', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = mount(<ProductView/>);
    });

    test('renders correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot(); 
    }); 
     
});