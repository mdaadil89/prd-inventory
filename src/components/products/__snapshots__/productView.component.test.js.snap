// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllProductsPage Snapshot renders correctly 1`] = `
<ProductView>
  <div
    style={
      Object {
        "justifyContent": "centre",
      }
    }
  >
    <Card
      body={false}
      border="primary"
      style={
        Object {
          "float": "none",
          "margin": "0 auto",
          "marginBottom": "10px",
          "width": "20rem",
        }
      }
    >
      <div
        className="card border-primary"
        style={
          Object {
            "float": "none",
            "margin": "0 auto",
            "marginBottom": "10px",
            "width": "20rem",
          }
        }
      >
        <CardHeader>
          <div
            className="card-header"
          >
            <b />
          </div>
        </CardHeader>
        <CardBody>
          <div
            className="card-body"
          >
            <br />
            <b />
            <br />
            <b>
              Made by 
            </b>
            <br />
            <b>
              Rs. 
            </b>
            <br />
            <b>
              Qty : 
            </b>
            <br />
          </div>
        </CardBody>
        <button
          className="btn btn-success"
          onClick={[Function]}
        >
          Go Back to List
        </button>
      </div>
    </Card>
  </div>
</ProductView>
`;
